Sources/Converters/Public/ExtenderConverter.swift

```diff
@@ -10,12 +10,45 @@ import Foundation
 public enum ExtenderConverter {
 	private static let maximumDuration: TimeInterval = 10
 
-	// swiftlint:disable line_length
 	private static let pattern =
 		#"""
```

Comment:
Out of curiosity, what does the `#` mean?

Reply:
It changes the escape character from `\` to `#`. This way, if you want to have a literal `\` in a string, you don't have to escape it with another `\`. See more [here](https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html#ID606).